meshers/blocky/voxel_library.h

```diff
@@ -5,6 +5,8 @@
 #include "voxel.h"
 #include <core/resource.h>
 
+#define NULL_INDEX 0xFFFFFFFF
```

Comment:
Use a `static const uint32_t`, not a macro

Reply:
Updated. Why is this way better?